CanonicalLtd/docs-landscape
# update header and menu

## Done

* used just the Landscape logo, not the word mark
* moved the release notes to a single file, below the On-Premises
* renamed the intro the index file

## QA

1. build the docs
2. see that the logo is just the logo, no wordmark
3. see that the initial file is now called index.html
